contrib/kafka/config/kafka.yaml

```diff
@@ -109,6 +109,11 @@ metadata:
 data:
   # Broker URL's for the provisioner. Replace this with the URL's for your kafka cluster.
   bootstrap_servers: kafkabroker.kafka:9092
+
+  # Consumer mode to dispatch events from different partitions in parallel.
+  # By default, partitions are multiplexed with a single go channel.
+  # `partitions` is the only valid value.
```

Comment:
Not exactly sure wat yoyu mean here.

See:
https://github.com/knative/eventing/blob/3fc44ec89a0ef13f56c0232dd4d986d5438be125/contrib/kafka/pkg/controller/util_test.go#L59-L68
and here:
https://github.com/knative/eventing/blob/3fc44ec89a0ef13f56c0232dd4d986d5438be125/contrib/kafka/pkg/dispatcher/dispatcher.go#L187-L190

Sarama does behave differently, with these two options: 

https://dev.to/davidsbond/golang-implementing-kafka-consumers-using-sarama-cluster-4fko

Reply:
Thank you for the comment.
I wanted to mean that any value (like `foo`, `bar` etc..) becomes `multiplex mode`, because current code handled only `partitions`. But I'm sorry my words did not explain it correctly.

https://github.com/knative/eventing/blob/3fc44ec89a0ef13f56c0232dd4d986d5438be125/contrib/kafka/pkg/controller/util.go#L44-L49

I added the code to handle the value `multiplex` (still `multiplex` is used by default) and updated the comments.